sdk/cosmos/azure_data_cosmos/src/options/mod.rs

```diff
@@ -1,9 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-use azure_core::http::{ClientMethodOptions, ClientOptions, Etag};
-
-use crate::models::ThroughputProperties;
+use crate::{
+    constants::{self},
+    models::ThroughputProperties,
+};
+use azure_core::http::headers::{AsHeaders, HeaderName, HeaderValue};
+use azure_core::http::{headers, ClientMethodOptions, ClientOptions, Etag};
+use serde_json::value::Index;
```

Comment:
You're not actually using this. Probably got brought in by VS Code when you were typing out `IndexingPolicy` somewhere :)

```suggestion
```